src/audio/mux/mux.c

```diff
@@ -453,6 +456,31 @@ static int mux_set_config(struct processing_module *mod, uint32_t config_id,
 				  fragment, fragment_size);
 }
 
+static int demux_trigger(struct processing_module *mod, int cmd)
```

Comment:
This bit is for mt8195, actually, not MTL (IPC4 topologies use "copier" components and not mux/demux).  And the xrun flags don't permit blocking really (unless it's a feature I'm unaware of), they toss an error during_pipeline copy().   This is the only spot I couldn't handle from within AEC itself, because component init() doesn't have access to the sink/source connections yet and the pipeline it's on may not see any trigger calls before the playback pipeline connected to it starts.

Reply:
ack, so mux is simpler IPC3 version of IPC4 copier, so changes here wont be runtime tested by any IPC4 CI (as there are no IPC4 topology constructs for it). I'm guessing we cant automatically set overflow allowed in topology ?